src/metabase/db/setup.clj

```diff
@@ -120,6 +124,33 @@
       (log/infof "Successfully verified %s %s application database connection. %s"
                  (.getDatabaseProductName metadata) (.getDatabaseProductVersion metadata) (u/emoji "✅")))))
 
+(mu/defn- check-encryption
+  "Ensure encryption env variable is correctly set if needed, and encrypt the database if it needs to be
+  Encryption status is tracked by an 'encryption-check' value in the settings table.
+  NOTE: the encryption-check setting is not managed like most settings with 'defsetting' so we can manage checking the raw values in the database"
+  [db-type :- :keyword
+   data-source :- (ms/InstanceOfClass javax.sql.DataSource)]
+  (when (or (liquibase/table-exists? "setting" (.getConnection ^javax.sql.DataSource data-source)) (liquibase/table-exists? "SETTING" (.getConnection ^javax.sql.DataSource data-source)))
```

Comment:
nit: this line's too long 

~I thought we had a linter for that.~

No, actually it's set in `dir-locals.el` which only works with emacs. Try to keep them [under 118 long](https://github.com/metabase/metabase/blob/master/.dir-locals.el#L6-L9)